### ISSUE TITLE
Lunar Feature: hide menu bar icon

### ISSUE THREAD

**Issue by DmitrySivkov** [alin23/Lunar#239]
### Name of the feature

"Hide menu bar icon" option.

### What problem does this feature solve?

Will make it possible to hide the menu bar icon for people with crowded menu bars.

### How should the feature work

A toggle button in the preferences menu or wherever.

**Comment by danilokleber** [comment]
We've got this in version 4 (thanks!) but I couldn't bring back the app window once the menu bar is hidden. The only way was killing the process from something like Activity Monitor and reopening Lunar. Are you aware of that?

**Comment by alin23** [comment]
Weird, there is code that should show the window when you launch the app. But I can see that it's not working now.

I'll fix it as soon as possible, thanks for the heads up!

**Comment by alin23** [comment]
Ok, I found the issue and fixed it. It will be released in 4.0.1 later today

**Comment by alin23** [comment]
Done in v4.0.2

**Comment by danilokleber** [comment]
I guess this is happening again as of 4.0.4 🤔

**Comment by alin23** [comment]
@danilokleber indeed, there was more to consider than I thought. 4.0.5 is already available and should fix this.

I also added an option to show Dock icon as sometimes Lunar doesn't receive any events if you only double-click on Lunar.app or launch it from something like Spotlight/Alfred.

If both the Dock icon and the menubar icon are hidden, it's better to use the **Show Lunar** global hotkey to be 100% sure the app shows its window.
